src/apim_checkout.tf

```diff
@@ -73,6 +81,113 @@ module "apim_checkout_payments_api_v1" {
+
+  name                  = format("%s-checkout-payment-activations-api", var.env_short)
+  api_management_name   = module.apim.name
+  resource_group_name   = azurerm_resource_group.rg_api.name
+  product_ids           = [module.apim_checkout_product[0].product_id]
+  subscription_required = local.apim_checkout_payment_activations_api.subscription_required
+  version_set_id        = azurerm_api_management_api_version_set.checkout_payment_activations_api[0].id
+  api_version           = "v1"
+  service_url           = local.apim_checkout_payment_activations_api.service_url
+
+  description  = local.apim_checkout_payment_activations_api.description
+  display_name = local.apim_checkout_payment_activations_api.display_name
+  path         = local.apim_checkout_payment_activations_api.path
+  protocols    = ["https"]
+
+  content_format = "swagger-json"
+  content_value = templatefile("./api/checkout/checkout_payment_activations/v1/_swagger.json.tpl", {
+    host = azurerm_api_management_custom_domain.api_custom_domain.proxy[0].host_name
+  })
+
+  xml_content = templatefile("./api/checkout/checkout_payment_activations/v1/_base_policy.xml.tpl", {
+    origin = format("https://%s.%s/", var.dns_zone_checkout, var.external_domain)
+  })
+}
+
+# pagopa-proxy SOAP web service FespCdService
+locals {
+  apim_cd_info_wisp = {
+    display_name          = "Checkout CdInfoWisp"
+    description           = "SOAP service used from Nodo to relay idPayment"
+    path                  = "api/checkout/CdInfoWisp"
+    subscription_required = false
+    service_url           = null
+  }
+}
+
+resource "azurerm_api_management_api_version_set" "cd_info_wisp" {
+  count = var.pagopa_proxy_enabled ? 1 : 0
+
+  name                = format("%s-cd-info-wisp", var.env_short)
+  resource_group_name = azurerm_resource_group.rg_api.name
+  api_management_name = module.apim.name
+  display_name        = local.apim_cd_info_wisp.display_name
+  versioning_scheme   = "Segment"
+}
+
+resource "azurerm_api_management_api" "apim_cd_info_wisp_v1" {
+  count = var.pagopa_proxy_enabled ? 1 : 0
+
+  name                  = format("%s-cd-info-wisp", var.env_short)
+  api_management_name   = module.apim.name
+  resource_group_name   = azurerm_resource_group.rg_api.name
+  subscription_required = local.apim_cd_info_wisp.subscription_required
+  service_url           = local.apim_cd_info_wisp.service_url
+  version_set_id        = azurerm_api_management_api_version_set.cd_info_wisp[0].id
+  version               = "v1"
+  revision              = "1"
+
+  description  = local.apim_cd_info_wisp.description
+  display_name = local.apim_cd_info_wisp.display_name
+  path         = local.apim_cd_info_wisp.path
+  protocols    = ["https"]
+
+  soap_pass_through = true
+
+  import {
+    content_format = "wsdl-link"
+    content_value  = "https://raw.githubusercontent.com/pagopa/io-pagopa-proxy/v0.20.1-RELEASE/src/wsdl/CdPerNodo.wsdl"
```

Comment:
why here reference to wsdl link instead of local file , as other soap import ?

Reply:
It was made more to make the file version explicit than anything else, but I can can include the file directly in the repo to make it consistent with the other wsdl definitions.